dcrpm/yum.py

```diff
@@ -31,12 +32,25 @@ def __init__(self):
         # type: () -> None
         self.logger = logging.getLogger()  # type: logging.Logger
         self.status_logger = logging.getLogger("status")  # type: logging.Logger
+        try:
+            self.yum = which(YUM_CMD_NAME)
+        except DcRPMException:
+            try:
+                self.yum = which(DNF_CMD_NAME)
+            except DcRPMException:
+                self.yum = None
+        if self.yum:
+            self.logger.info("Using %s for yum" % self.yum)
+        else:
+            self.logger.error("Cannot find yum!")
```

Comment:
shouldn't this exit? or raise?

Reply:
Eventually, yes, but for now I'm trying to keep the existing behavior as close as possible.

Reply:
but that's my point. The existing behavior would try to run chef, throw a stacktrace and fail. You're changing the behavior drastically: it will log something but exit with success :(

Reply:
Oh I see what you mean now, yeah you're right